cmd/cli/telegram_utils.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis"
+	"github.com/moira-alert/moira/senders/telegram"
+)
+
+var (
+	telegramUsersKey = "moira-telegram-users:"
+	telegramLockName = "moira-telegram-users:moira-bot-host"
+)
+
+func updateTelegramUsersRecords(logger moira.Logger, database moira.Database) error {
+	logger.Info().Msg("Start updateTelegramUsersRecords")
+
+	switch d := database.(type) {
+	case *redis.DbConnector:
+		pipe := d.Client().TxPipeline()
+		iter := d.Client().Scan(d.Context(), 0, telegramUsersKey+"*", 0).Iterator()
+		for iter.Next(d.Context()) {
+			key := iter.Val()
+			if key == telegramLockName {
```

Comment:
In a recent PR #1027 we changed the key name slightly from `moira-telegram-users:moira-bot-host` to `moira-telegram-users:moira-bot-host:*`, so I suggest changing the code like this:
```
var (
	telegramUsersKey = "moira-telegram-users:"
	telegramLockName = "moira-telegram-users:moira-bot-host:"
)

if strings.HasPrefix(key, telegramLockName) {
    continue
}

```

Reply:
Done in https://github.com/moira-alert/moira/commit/ff21de6760d0f4ca4df9f993a0288289d2b54a91